aip/auth/4110.md

```diff
@@ -147,13 +149,14 @@ digraph d_front_back {
   1. Use self-signed JWT flow to create an access token locally.
     1. If certificate is presented, embed the certificate into the JWT.
     1. Use the regular [self-signed JWT flow][4] for an access token. _[END]_
-  1. Use user identity flow to exchange for an access token. _[END]_
+  1. Use the gcloud default credential to exchange for an access token. _[END]_
```

Comment:
Can you link this flow to 4113? Something like:

Use the [gcloud default credential to exchange](./4113.md#expected_behaviors) for an access token. _[END]_